packages/cli/generators/app/index.js

```diff
@@ -49,6 +49,11 @@ module.exports = class AppGenerator extends ProjectGenerator {
       description: g.f('Include service-proxy imports and ServiceMixin'),
     });
 
+    this.option('apiconnect', {
+      type: Boolean,
+      description: g.f('Include ApiConnectComponent'),
```

Comment:
I'm thinking whether we can make the text more user-friendly. something like `Include ApiConnectComponent to work with IBM API Connect` or `Include ApiConnectComponent in Application class`?

Reply:
I used the similar description as other features. :-)